osidb/models.py

```diff
@@ -1291,6 +1319,11 @@ def fix_state(self):
                 return Affect.AffectFix.DEFER
         return Affect.AffectFix.AFFECTED
 
+    @property
+    def is_closed(self):
+        if self.status:
+            return self.status.upper() == "CLOSED"
```

Comment:
Not sure whether this will cover all cases -- if SFM2 does it like this then OK, but there are definitely JIRA end states that are not called `CLOSED`

Reply:
It is how (SFM2 defined)[https://git.prodsec.redhat.com/devops/sfm2/-/blob/master/sfm2/dmodels/abstract/__init__.py#L344] it, but I can do further implementation, how I can find out JIRA statuses options?

Reply:
It is a wild west there but as this is the SFM2 implementation and nobody complains I would stick with it for now.

Reply:
Yep we should only care about feature parity right now which is why I asked, so we can keep this as-is, we can improve it in the future if there's an actual need.